src/modules/logger/logger.h

```diff
@@ -55,8 +55,11 @@
 
 extern "C" __EXPORT int logger_main(int argc, char *argv[]);
 
-static constexpr hrt_abstime TRY_SUBSCRIBE_INTERVAL{1000 * 1000};	// interval in microseconds at which we try to subscribe to a topic
-// if we haven't succeeded before
+#ifdef ORB_USE_PUBLISHER_RULES
+static constexpr hrt_abstime TRY_SUBSCRIBE_INTERVAL {1000};
+#else
+static constexpr hrt_abstime TRY_SUBSCRIBE_INTERVAL {1000 * 1000}; // interval in microseconds at which we try to subscribe to a topic
+#endif
```

Comment:
Why do we need that?

Reply:
Removed for now. Might be less relevant after https://github.com/PX4/Firmware/pull/14106. I'll readd this (or solve it differently) if needed in master.